test/modules/aws-codebuild-integration.ts

```diff
@@ -361,13 +361,87 @@ phases:
   );
 
   it('starts a transaction', begin());
+  it(
+    'creates a project that pushes to ecr',
+    query(
```

Comment:
you should probably just use the `generate_put_ecr_image_build_spec` function here?

Reply:
If that is used elsewhere in the test suite, then I would say it's fine. Better to exercise more possible paths.

Reply:
It's "building" a specific folder. In this case I've just pulled an image, retagged and pushed (to reduce the rate of errors). For that case we'll need to add Github credentials which will test multiple things.
If that's not a blocker, I'd suggest we go with the current version.

Reply:
that makes sense 👍